src/test/dependencies.test.ts

```diff
@@ -65,6 +68,47 @@ suite("dependencies", () => {
             listener => myDependency.install("remote", true, listener));
     });
 
+    test("Get tagged asset url", async function() {
+        this.timeout(PRUSTI_UBUNTU_ASSET_DOWNLOAD_TIMEOUT_MS);
+        const assetName = "prusti-release-ubuntu.zip";
+        const url = await GitHubReleaseAsset.getTaggedAssetUrl(
+            "viperproject", "prusti-dev", assetName, "v-2020-10-06-1807");
+        console.info(`URL to prusti-dev Ubuntu asset of v-2020-10-06-1807 release: '${url}'`);
+
+        // now download the zip:
+        const headers = {
+            "Accept": "application/octet-stream"
+        };
+        const myDependency = new Dependency<"remote">(
+            TMP_PATH,
+            ["remote",
+                new InstallerSequence([
+                    new FileDownloader(url, headers, assetName)
+                ])
+            ]
+        );
+        const { result: downloadDestination } = await withProgressInWindow(
+            "Testing download of a GitHub asset",
+            listener => myDependency.install("remote", true, listener));
+        console.log(`Prusti asset downloaded to '${downloadDestination.basePath}'`);
```

Comment:
I would remove this and all other `console.log` in the tests, to keep the output clean.